app/controllers/import_controller.rb

```diff
@@ -60,10 +60,12 @@ def import
               elsif col_num == 85 && workflow == :isolation
                 patient[:user_defined_symptom_onset] = row[85].present?
                 patient[field] = validate_field(field, row[col_num], row_ind)
-              elsif col_num == 86
-                patient[field] = validate_workflow_specific_enums(workflow, field, row[col_num], row_ind)
+              # TODO: Commented out temporarily until we're comfortable with this validation
+              # elsif col_num == 86
+              #   patient[field] = validate_workflow_specific_enums(workflow, field, row[col_num], row_ind)
               else
-                patient[field] = validate_field(field, row[col_num], row_ind) unless col_num == 85 && workflow != :isolation
+                 # TODO: Once the above is uncommented, this line can be updated to not have to check the 86 col
```

Comment:
really minor thing, but it looks like there's an extra space before the comment here

Reply:
I think rubocop added that oddly, I'll remove if not.

Reply:
More like - I fixed it locally with rubocop and never pushed haha. Should be fine now.

Reply:
More like - I fixed it locally with rubocop and never pushed haha. Should be fine now.